lib/ansible/executor/module_common.py

```diff
@@ -92,7 +92,22 @@
 # LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 import os
+import os.path
 import sys
+import __main__
+
+# With Python3 on Ubuntu, we pick up this script in the temporary directory.
```

Comment:
All Ubuntu versions or only certain ones? That will make it clearer if someone needs to update this in the future to know what needs testing